pyproject.toml

```diff
@@ -85,6 +85,11 @@ servicex = [
   "servicex>=2.5.3",
   "func-adl_servicex",
 ]
+rucio = [
+  "rucio-clients>=32;python_version>'3.8'",
+  "rucio-clients<32;python_version<'3.9'",
+  "cmd2",
```

Comment:
Just to check, is some of the interactive console stuff available via `rich` (already a dependency) or do we need `cmd2`?

Reply:
@valsdav

Reply:
switched to full rich cli, removed cmd2 dependency